### PR TITLE
Actualizacion - conexion con BD

### PR DESCRIPTION
**Pull request by erickfabiandev**

### CODE REVIEW

File: src/pages/Admin/Admin.jsx [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+import Doctors from '../../components/Admin/Doctors/Doctors'
+import axios from 'axios'
+import host from '../../services/host'
+import { useLoaderData } from 'react-router'
+
+const Admin = () => {
+  const fullName = localStorage.getItem('fullName')
+  const { usersData } = useLoaderData()
+  const doctorsData = usersData.filter((item) => { return item.doctor })
+  console.log(doctorsData)
```
Comment (ht1204) [inline src/pages/Admin/Admin.jsx:13]: is this console.log necessary?

File: src/pages/App.jsx [hunk 2]
```diff
@@ -1,41 +1,16 @@
...
 
 import UserContext from '../context/UserContext'
 import CartProductsContext from '../context/CartProductsContext'
 // Temporary
 import cartProducts from '../__mocks__/cart-products.json'
 import './App.scss'
 
-function App () {
+function App() {
   // TODO: set the productList initial value to an empty [] array after implementing the Cart component
```
Comment (ht1204) [inline src/pages/App.jsx:14]: is this change done? Why is still this comment?

File: src/pages/Home/Home.jsx [hunk 3]
```diff
@@ -11,23 +11,25 @@ import DoctorCarousel from '../../components/DoctorCarousel/DoctorCarousel'
 import OurPartners from '../../components/OurPartners/OurPartners'
 import LatestNews from '../../components/LatestNews/LatestNews'
 import Faq from '../../components/FAQ/Faq'
+import FirstViewUnlogged from '../../components/UnloggedMainView/FirstViewUnlogged'
 // faq modification
 
 const Home = () => {
+  const ROLE = localStorage.getItem('role')
```
Comment (ht1204) [inline src/pages/Home/Home.jsx:18]: why is this info in localStorage? This is a security flaw.

File: src/pages/Login/Login.jsx [hunk 4]
```diff
@@ -37,38 +71,48 @@ function Login () {
...
+        pageTitle={'Login'}
+        backgroundClassName="login__bg"
+        breadcrumb={breadcrumb}
+      />
+      <div className="login-container">
+        {
+          loading &&
+          <img
+            src='https://res.cloudinary.com/dzmkilinu/image/upload/v1692693479/medical-site/loading_xcwewp.gif'
+            style={{ width: '200px', height: '200px', textAlign: 'center' }}
```
Comment (ht1204) [inline src/pages/Login/Login.jsx:85]: I suggest the image link could be better in an importable constant.

File: src/pages/Login/Login.jsx [hunk 5]
```diff
@@ -37,38 +71,48 @@ function Login () {
...
+          />
+        }
+        {
+          !loading && <>
```
Comment (ht1204) [inline src/pages/Login/Login.jsx:89]: In JSX, it's better for conditional rendering surround the component with parenthesis.

File: src/router/ProtectedRoute.jsx [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+export const ProtectedRoute = (
+  {
+    authorizedRoles,
+    redirectTo = '/',
+    children
+  }
+) => {
+  const ROLE = localStorage.getItem('role')
+  const TOKEN = localStorage.getItem('token')
+  const STATUS = localStorage.getItem('status')
```
Comment (ht1204) [inline src/router/ProtectedRoute.jsx:15]: I suggest refactor the info management.

**Review by ht1204 — CHANGES_REQUESTED** [review]
Please address the comments,  provide the info of the purpose of this PR, I'm attentive to any enhancement and change. Thx.

**Review by andrystylist — APPROVED** [review]

**Review by ht1204 — APPROVED** [review]

File: src/pages/Home/Home.jsx (hunk 3 above)
Comment (erickfabiandev) [reply src/pages/Home/Home.jsx:18]: it will be updated in a new PR

**Pull request merged**